src/providers/pkcs11/key_management.rs

```diff
@@ -95,6 +95,11 @@ impl Provider {
         let key_name = op.key_name;
         let key_attributes = op.attributes;
 
+        if key_attributes.policy.usage_flags.export && !self.allow_export {
+            error!("The configuration of this provider does not allow it to generate keys that can be exported.");
+            return Err(ResponseStatus::PsaErrorNotPermitted);
+        }
```

Comment:
Should this be in `psa_import_key_internal` too? We don't handle private keys for now, though.

Reply:
I think ultimately it should be yes! I did not add it now for that reason.

Reply:
Well, added it anyway 🌞